provider/icu4x-datagen/src/main.rs

```diff
@@ -295,16 +288,26 @@ fn main() -> eyre::Result<()> {
     let markers = if !cli.markers.is_empty() {
         match cli.markers.as_slice() {
             [x] if x == "none" => Default::default(),
-            [x] if x == "all" => icu_datagen::all_markers(),
+            [x] if x == "all" => {
+                #[cfg(feature = "experimental")]
+                log::warn!("The icu4x-datagen crate has been built with the `experimental` feature, so `--markers all` includes experimental markers");
+                #[cfg(not(feature = "experimental"))]
+                log::warn!("The icu4x-datagen crate has been built without the `experimental` feature, so `--markers all` does not include experimental markers");
```

Comment:
It's not really a warning, is it?

```suggestion
                #[cfg(feature = "experimental")]
                log::info!("The icu4x-datagen crate has been built with the `experimental` feature, so `--markers all` includes experimental markers");
                #[cfg(not(feature = "experimental"))]
                log::info!("The icu4x-datagen crate has been built without the `experimental` feature, so `--markers all` does not include experimental markers");
```